lua/hurl/main.lua

```diff
@@ -190,6 +192,10 @@ local function execute_hurl_cmd(opts, callback)
 
   -- Include the HTTP headers in the output and do not colorize output.
   local cmd = vim.list_extend({ 'hurl', '-i', '--no-color' }, opts)
+  if is_file_mode then
+    local file_root = _HURL_GLOBAL_CONFIG.file_root or vim.fn.getcwd()
```

Comment:
Could you add the to global variable about the new config? (`file_root`) @Xouzoura